doc/attr.xml

```diff
@@ -1439,6 +1439,42 @@ gap> ChromaticNumber(D);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="ChromaticNumberFunc">
+<ManSection>
+    <Func Name="ChromaticNumber" Arg="digraph, colouring algorithm" 
```

Comment:
The argument name with a space is misleading - perhaps use an underscore?

Reply:
Thanks, I've changed that